ci/functional/job_cleanup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Common cleanup for running after a functional test
+
+set -uex
+
+rm -rf install/lib/daos/TESTING/ftest/avocado/job-results/job-*/html/
+
+# Remove the latest avocado symlink directory to avoid inclusion in the
+# jenkins build artifacts
+unlink install/lib/daos/TESTING/ftest/avocado/job-results/latest
+rm -rf "Functional/"
+mkdir "Functional/"
+# compress those potentially huge DAOS logs
+if daos_logs=$(find \
+    install/lib/daos/TESTING/ftest/avocado/job-results/job-*/daos_logs/* \
+    -maxdepth 0 -type f -size +1M); then
+  # shellcheck disable=SC2086
+  lbzip2 $daos_logs
+fi
+  # shellcheck disable=sc2035
```

Comment:
(lint) Couldn't parse this shellcheck annotation. [SC1073]
(lint) Expected "\n". Fix any mentioned problems and try again. [SC1072]